connect/cli/plugins/product/export.py

```diff
@@ -402,7 +402,7 @@ def _fill_item_row(ws, row_idx, item):
     ws.cell(row_idx, 5, value=item['description'])
     ws.cell(row_idx, 6, value=item['type'])
     ws.cell(row_idx, 7, value=item['precision'])
-    ws.cell(row_idx, 8, value=item['unit']['unit'])
+    ws.cell(row_idx, 8, value=item['unit']['id'])
```

Comment:
You cannot change the column on export, it will break compatibility.

You should modify the sync and make a lookup of the id from the name.